Documentation/op-guide/grpc_proxy.md

```diff
@@ -98,3 +98,43 @@ $ ETCDCTL_API=3 ./etcdctl --endpoints=127.0.0.1:2379 get foo
 foo
 bar
 ```
+
+## Client endpoint synchronization and name resolution
+
+The proxy supports registering its endpoints for discovery by writing to a user-defined endpoint. This serves two purposes. First, it allows clients to synchronize their endpoints against a set of proxy endpoints for high availability. Second, it is an endpoint provider for etcd [gRPC naming][dev-guide/grpc_naming.md].
+
+Register proxy by providing a user-defined prefix:
+
+```bash
+$ etcd grpc-proxy start --endpoints... \
+  --listen-addr=10.240.0.1:23790 \
+  --advertise-client-url=10.240.0.1:23790 \
+  --resolver-prefix="___grpc_proxy_endpoint/" \
+  --resolver-ttl=60
+```
+
+The proxy will list all its members for member list:
+
+```bash
+ETCDCTL_API=3 ./bin/etcdctl --endpoints=http://10.240.0.1:23790 member list
+
+0, unstarted, , , 127.0.0.1:23790
```

Comment:
should the proxy member list return a valid version? otherwise it'll fail strict version checking